symbiflow-vtr-optimized/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v','') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN')|replace('-','_') %}
+
+package:
+  name: symbiflow-vtr-optimized
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/SymbiFlow/vtr-verilog-to-routing.git
+    git_rev: f1a3bcc2a8aa93d88cb26ca7f1853962fff57c32
```

Comment:
Why are you using a git hash instead of a branch name?

Reply:
This is the same hash used in symbiflow-arch-defs:
https://github.com/SymbiFlow/symbiflow-arch-defs/blob/991c74f0e1d579f143878b22d72ab65235f3c18c/environment.yml#L9

`master+wip` doesn't work. Would you like me to create a `master+wip-stable` branch?

Reply:
Why doesn't `master+wip` work?  @acomodi PR here was green: https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1711

Reply:
It doesn't work in this test: https://travis-ci.com/github/SymbiFlow/conda-packages/jobs/441888559

Reply:
Here's a test with the architecture definition files from https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1711: https://travis-ci.com/github/SymbiFlow/conda-packages/jobs/441905520

Reply:
So fix it?  Or an absolute minimum, file a bug?  If you believe you have a reproducible error caused by https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1711 , that is something that needs to be resolved.

Reply:
> https://travis-ci.com/github/SymbiFlow/conda-packages/jobs/441905520

This test failure is a result of using a eblif generated from an earlier version of arch-defs.  This highlights the fragility of using a frozen eblif.  What is your procedure for regenerating the eblif when arch-defs is updated?

Reply:
I think there is some confusion. The bug is in SymbiFlow/vtr-verilog-to-routing master+wip, which I think is indicated by the SymbiFlow+VtR tests that we know to be failing.

https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1711 uses the same version of [SymbiFlow/vtr-verilog-to-routing](https://github.com/SymbiFlow/vtr-verilog-to-routing/commit/f1a3bcc2a8aa93d88cb26ca7f1853962fff57c32) that the latest (including #1711) symbiflow-arch-defs uses.

Reply:
So the link you gave for the 1711 CI result failure was likely caused because the eblif was out of sync, specifically:

```
Type: Blif file
File: top.eblif
Line: 59289
Message: Found no matching port 'CO_CHAIN' on architecture model 'CARRY4_VPR'
```

Reply:
The reason this hasn't come up before is that building the Conda package does not run any tests with an xc7 architecture; adding these tests would likely cause the other symbiflow-vtr-* packages to fail as well.

Reply:
And for reference, `symbiflow+vtr` from #1711 was green: https://source.cloud.google.com/results/invocations/bc061b4a-3fb7-447b-9c3f-8237ecbed644/targets

Reply:
> The reason this hasn't come up before is that building the Conda package does not run any tests with an xc7 architecture; adding these tests would likely cause the other symbiflow-vtr-* packages to fail as well.

Totally agree with the above statement.  So what is your answer to solving this problem?

Reply:
I will add a test to build.sh once this is in: https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/1564

Reply:
Yes, I need to get a new eblif from the latest symbiflow-arch-defs.

Reply:
I've tried an eblif built from https://github.com/SymbiFlow/symbiflow-arch-defs/commit/d299ba7792fc4d3ccd9113b1629ccb21d877e7dd with https://github.com/SymbiFlow/vtr-verilog-to-routing/commit/1243707ceacab205ca49c58c10d004590fdaf9b8, the same versions as used in https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1711 from the VtR+SymbiFlow build log, and it still won't build. I suspect a difference in the `symbiflow_*` scripts, so I may need to copy the exact commands used from the symbiflow-arch-defs build.

Reply:
I may be able to get fpga-tool-perf working now, so I'll use the eblif from that instead.

Reply:
I now have this working with master+wip.